### PR TITLE
Fz12 ok 3 sign up

### PR DESCRIPTION
**Pull request by kamiljaworski975**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,9 +72,30 @@ <h2>Fast packing - easy travel</h2>
         </section>
 
     </section>
-
+<!-- Kamil Jaworski-->
         <section id="sign-up" class="sign-up">
-            
+            <section id="sign-up" class="sign-up">
```
Comment (adambrudzynski) [inline index.html]: doubled sections with same ID

File: index.html [hunk 2]
```diff
@@ -72,9 +72,30 @@ <h2>Fast packing - easy travel</h2>
...
+                <div class="ourrow">
+                    <div class="sign-text">
+                    <h2>Sign up</h2>
+                    <h3>We would love to join you with your next travel expirance.
```
Comment (adambrudzynski) [inline index.html]: spelling: "experience"

**Review by adambrudzynski — CHANGES_REQUESTED** [review]
comments in the code

File: style.css [hunk 3]
```diff
@@ -87,6 +87,60 @@ h2:after {
...
+  }
+  
+  .submit-btn:hover,
+  .submit-btn:active {
+    background-color: #bf6516;
+    border-radius: 4px;
+    border: none;
+  }
+  
+  .container {
```
Comment (Makrenek) [inline style.css]: ? check if it dosent double in bootstrap

**Review by Makrenek — CHANGES_REQUESTED** [review]

**Review by adambrudzynski — APPROVED** [review]

**Review by Makrenek — APPROVED** [review]

**Pull request merged**